src/human_player.c

```diff
@@ -48,13 +49,26 @@ int request_card(PlayerType_e PlayerType)
     //Assigning the drawn card to the human player
     return add_card_at_end(g_players[HUMAN].cards_on_hand, card);
 }
+
 /**
  * @brief Throws a warning message if the card dropped by the user is invalid
  * 
  */
 void invalid_card_warning(void)
 {
-    printf("!!Warning!! Invalid card - Please choose a valid card \n");
+    printf("!!Warning!! Invalid card - Please choose a valid card. \n");
+    display_player_deck(HUMAN);
+
+    return;
+}
+
+/**
+ * @brief Throws a warning message if the card dropped by the user is unplayable
+ *
+ */
+void unplayable_card_warning(void)
```

Comment:
Can you please change this as invalid_choice_warning? unplayable gives a different meaning..

Reply:
Can you be specific please? We have invalid warning now.

Reply:
Invalid warning (which we have already) means the card  value entered by the user during discard action is invalid- which can be because the card entered is of an invalid format.
My concern is that when you say it's an unplayable card- it might mean that the card cannot be played at all. That's why I suggested to change it to as "invalid_choice_warning" - this will mean the choice during that turn is an invalid one. Hope that clears it.

Reply:
I will leave it as is. I removed this function.